esmvaltool/diag_scripts/gwls/plot_gwl_exceedance_mm_stats.py

```diff
@@ -0,0 +1,276 @@
+        # select by by project
+        if data["project"] not in projects:
+            projects.append(data["project"])
+
+    logger.info("List of Projects: %s", projects)
+
+    for project in projects:
+        # get all data sets for project across experiments
+        project_data = select_metadata(input_data, project=project)
+        for gwl in gwls:
+            gwl_subset_df = gwl_df[gwl_df["GWL"] == gwl]
+            if gwl_subset_df.shape[0] > 0:
+                logger.info(
+                    "Calculating means and standard deviations for GWL: %s",
+                    str(gwl),
+                )
+                filename = sep.join([project, "mm_mean", str(gwl)]) + ".nc"
+                mean_file = os.path.join(cfg["work_dir"], filename)
+                filename = sep.join([project, "mm_stdev", str(gwl)]) + ".nc"
+                stdev_file = os.path.join(cfg["work_dir"], filename)
+
+                cubes = calculate_gwl_mm_cube(
+                    project_data,
+                    gwl_subset_df,
+                    window_size,
+                )
+                calculate_mm_stats(cubes, mean_file, stdev_file)
+                filename = sep.join([project, "mm_mean", str(gwl)]) + ".png"
+                mean_plot_file = os.path.join(cfg["plot_dir"], filename)
+                filename = sep.join([project, "mm_stdev", str(gwl)]) + ".png"
+                stdev_plot_file = os.path.join(cfg["plot_dir"], filename)
+
+                # plot mean and stdev if they exist
+                if os.path.isfile(mean_file):
+                    logger.info("Plotting mm mean file %s", mean_file)
+                    cube = iris.load_cube(mean_file)
+                    plt.axes(projection=ccrs.Robinson())
+                    levels = np.arange(
+                        float(mean_level_params[0]),
+                        float(mean_level_params[1]),
+                        float(mean_level_params[2]),
+                    )
+                    if cfg["quickplot"]["title_var"] == "Temperature":
+                        qplt.contourf(
+                            cube,
+                            norm=colors.CenteredNorm(),
+                            cmap=cmap_mean_str,
+                            levels=levels,
+                            extend="max",
+                        )
+
+                    if cfg["quickplot"]["title_var"] == "Precipitation":
+                        qplt.contourf(
+                            cube,
+                            cmap=cmap_mean_str,
+                            levels=levels,
+                            extend="max",
+                        )
+
+                    plt.gca().coastlines()
+                    title_str = (
+                        "Multimodel mean of "
+                        + cfg["quickplot"]["title_var"]
+                        + " at "
+                        + str(gwl)
+                        + r"$^\circ$ C"
+                    )
```

Comment:
```suggestion
                    title_str = f"Multimodel mean of {cfg["quickplot"]["title_var"]} at {gwl} $^\\circ$ C"
```
You could use [f-strings](https://docs.python.org/3/tutorial/inputoutput.html#formatted-string-literals) for increased readability

Reply:
Will make the change.